src/components/blocks/table/Tr.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { ComponentProps } from 'react';
+import { HtmlComponentProps, ValidReactElement } from 'src/components/html-component-props';
+import Html from 'src/components/blocks/Html';
+import HtmlDataTypes from '../../../../data/types/html';
+
+type Props = { data: HtmlComponentProps<ValidReactElement>[]; attribs: ComponentProps<'tr'> };
+
+const Tr = ({ data, attribs }: Props) => (
```

Comment:
I think this should be `HtmlComponentProps<'tr'>`, and this time if data isString or is null, return <tr>{data}</tr>